src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1068,11 +1086,14 @@ bool PolicyHandler::ReceiveMessageFromSDK(const std::string& file,
                                           const BinaryMessage& pt_string) {
   POLICY_LIB_CHECK(false);
 
-  bool ret = policy_manager_->LoadPT(file, pt_string);
+  bool ret = CallPolicyManagerFunction(&PolicyManager::LoadPT, file, pt_string);
```

Comment:
@SNiukalov Const?

Reply:
@AByzhynar 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/2614/commits/98dc17dda0378940c016a6b9a16f1e3992ee8e65